### ISSUE TITLE
Input of RNN/LSTM

### ISSUE THREAD

**Issue by ksarker1** [keras-team/keras#8081]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by ksarker1** [comment]
Hi! I am new to ML. I am confused about the input of RNN/LSTM. I have a problem, where I want to feed 100 frames of size 17x2 in RNN/LSTM and there can be many of these sets, say batch size is 200. Is there any ways to do that? What I have tried is this below. It works as a 100 different frames. But I only have one label for that. And I have many such 100 frames to train. Any help would be great! Thank you in advance. 
model = Sequential()
model.add(LSTM(100,
               input_shape=(17, 2),
               batch_size=batch_size,
               return_sequences=True,
               stateful=True))
model.add(LSTM(1,
               return_sequences=False,
               stateful=True))
model.add(Dense(1))
model.compile(loss='mse', optimizer='rmsprop')

**Comment by unrealwill** [comment]
Hello, 
You should reshape your (17,2) frame into (34,)frame and have input_dim=34 (required) input_length=100 (optional)

**Comment by ksarker1** [comment]
Thank you very much, unrealwill. I will try that. I have coordinates (x,y). Would that mean the same?
